api/src/main/java/io/strimzi/api/kafka/KafkaConnectorList.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2018, Strimzi authors.
```

Comment:
2019. Tbh should we remove the year to avoid this kind of "issues"?

Reply:
I agree, we should remove the year, but we should so that across all source files in another PR.

Reply:
Agree. We should even change the `java.header` which is used for the header check.

Reply:
thanks @ppatierno done here https://github.com/strimzi/strimzi-kafka-operator/pull/2087